.ado/templates/apple-xcode-build.yml

```diff
@@ -10,19 +10,10 @@ parameters:
   xcode_extraArgs: ''
 
 steps:
-  # Install node_modules necessary for the project
-  - bash: |
-      echo "Running: cd apps/apple/FluentUITester/$platform_directory && yarn install" &&
-      cd apps/apple/FluentUITester/$platform_directory &&
-      yarn
-    env:
-      platform_directory: ${{ parameters.relative_directory }}
-    displayName: "yarn"
-
   # Install any pods necessary for the project
   - bash: |
-      echo "Running: cd apps/apple/FluentUITester/$platform_directory && pod install" &&
-      cd apps/apple/FluentUITester/$platform_directory &&
+      echo "Running: cd apps/apple/$platform_directory && pod install" &&
```

Comment:
Side note: we could also run `pod install --project-directory=$platform_directory/` if we want to save one command

Reply:
That makes a follow-up change to remove the apple directory easier too?

Reply:
Anand are you saying you want to split ios and macos into separate folders with separate package.json's?

Reply:
I thought that's what you and Saad recommended in side Teams chat?

Reply:
Okay cool yea I think it'll be better organized that way and give us parity with other platforms

Reply:
Yeah, that's the approach I thought we landed on too. I figured we would do that change after we had Lerna working, but doing them both at the same time is also great.
Side note: We can have a separate PR to delete `apps/mobile/ios` after this goes in?

Reply:
Will follow up with that fix since it's broken CI twice and can be easily worked on in a followup PR